frontend/src/app/features/work-packages/components/wp-card-view/wp-single-card/wp-single-card.component.sass

```diff
@@ -16,29 +17,30 @@
   &:hover
     .op-wp-single-card--inline-buttons
       opacity: 1
-
-  :not(&_closed):not(&_selected)
-    &:hover
-      .op-wp-single-card--card-actions
-        background-image: linear-gradient(to left, rgba(white,1), rgba(white,0))
+      z-index: 2
+      background-image: linear-gradient(to left, rgba($spot-color-basic-white,1), rgba($spot-color-basic-white,0))
 
   &_new
     padding-right: 25px
 
   &_selected
-    background-color: var(--table-row-highlighting-color)
+    background-color: $spot-color-main-light
+    op-icon
+      background: $spot-color-main-light
     &:hover
       .op-wp-single-card--inline-buttons
-        background-image: linear-gradient(to left, rgba(var(--table-row-highlighting-color),1), rgba(var(--table-row-highlighting-color),0))
+        background-image: linear-gradient(to left, rgba($spot-color-main-light,1), rgba($spot-color-main-light,0))
 
   &_disabled
     opacity: 0.6
 
   &_closed:not(&_selected)
-    background-color: #F3F3F3
+    background-color: $spot-color-basic-gray-6
+    op-icon
```

Comment:
Please do not use tags as selector. Instead use classes. If the element does not have an appropriate class, you can add one.